src/Database/LSMTree/Monoidal.hs

```diff
@@ -584,14 +584,14 @@ snapshot snap (Internal.MonoidalTable t) =
 -- @
 -- example session = do
 --   t <- 'new' \@IO \@Int \@Int \@Int session _
---   'snapshot' "intTable" t
---   'open' \@IO \@Bool \@Bool \@Bool session "intTable"
+--   'createSnapshot' "intTable" t
+--   'openSnapshot' \@IO \@Bool \@Bool \@Bool session "intTable"
 -- @
 --
 -- TOREMOVE: before snapshots are implemented, the snapshot name should be ignored.
 -- Instead, this function should open a table from files that exist in
 -- the session's directory.
```

Comment:
Could you remove this bit? Seems stale, even with our temporary implementation place it is using the snapshot name